### ISSUE TITLE
Option to set DSP Rate bar refresh frequency?

### ISSUE THREAD

**Issue by mxmilkiib** [falkTX/Carla#1482]
I'd be happy with a 1 second rate myself.

**Comment by falkTX** [comment]
this is already there no? the GUI refresh rate

**Comment by mxmilkiib** [comment]
Does that not also affect inline displays and other animated graphics?

**Comment by falkTX** [comment]
yes. there is a rate between the 2. the gui rate is what you set, it will animate the fast elements.
the "slow" elements update 5x slower than that IIRC
